tahoma_api/client.py

```diff
@@ -24,75 +26,55 @@ def __init__(self, username: str, password: str, api_url: str = API_URL) -> None
         self.password = password
         self.api_url = api_url
 
-        self.__cookies = None
         self.devices: List[Device] = []
         self.__roles = None
 
-    async def login(self) -> bool:
-
-        payload = {"userId": self.username, "userPassword": self.password}
-
-        async with aiohttp.ClientSession() as session:
-            async with session.post(self.api_url + "login", data=payload) as response:
-
-                result = await response.json()
-
-                # 401
-                # {'errorCode': 'AUTHENTICATION_ERROR',
-                #  'error': 'Bad credentials'}
-                # {'errorCode': 'AUTHENTICATION_ERROR',
-                #  'error': 'Your setup cannot be accessed through this application'}
-                if response.status == 401:
-                    if result["errorCode"] == "AUTHENTICATION_ERROR":
-
-                        if "Too many requests" in result["error"]:
-                            print(result["error"])
-                            raise Exception
-
-                        if (
-                            "Your setup cannot be accessed through this application"
-                            in result["error"]
-                        ):
-                            print(result["error"])
-
-                        if "Bad credentials" in result["error"]:
-                            print(result["error"])
-
-                        print(result["error"])
+        self.session = aiohttp.ClientSession(raise_for_status=False)
```

Comment:
raise_for_status is False by default: https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.request